src/commands/runDraftTool/helper/draftCommandBuilder.ts

```diff
@@ -83,6 +84,12 @@ export function buildCreateConfig(
       });
    }
 
+   if (imageTag.length > 0) {
```

Comment:
good point these are variables that come from draft, and can be programmatically determined from the `draft info` command json output. this could probably just be a separate PR